resources/swagger.json

```diff
@@ -55,6 +55,49 @@
         }
       }
     },
+    "/users/search": {
+      "get": {
```

Comment:
it should be a post

Also, we are receiving a body in the post, it should be in the params list as well